app/styles/pages/public.scss

```diff
@@ -12,3 +16,9 @@
   display: flex;
   flex-wrap: wrap;
 }
+
+@media screen and (min-width: 768px) and (max-width: 992px) {
+  .public-container {
+    padding: 0 .8rem;
+  }
+}
```

Comment:
Not correct

Reply:
Sir, may you please clarify that the bug is not fixed or the way of doing is the issue?

Reply:
Way of doing